addons/mail/static/src/views/fields/properties_field/property_value.js

```diff
@@ -0,0 +1,46 @@
+/** @odoo-module **/
+
+import { PropertyValue } from "@web/views/fields/properties/property_value";
+import { useOpenChat } from "@mail/web/open_chat_hook";
+import { patch } from "@web/core/utils/patch";
+import { TagsList } from "@web/views/fields/many2many_tags/tags_list";
+
+// Many2one
+patch(PropertyValue.prototype, "mail.PropertyValue", {
+    setup() {
+        this._super();
+        if (this.env.services.messaging) {
+            // work only for the res.users model
+            this.openChat = useOpenChat("res.users");
+        }
+    },
+
+    onAvatarClick() {
+        if (this.openChat && this.showAvatar && this.props.comodel === "res.users") {
+            this.openChat(this.props.value[0]);
+        }
+    },
+});
+
+// Many2many
+export class Many2manyPropertiesTagsList extends TagsList {
+    setup() {
+        super.setup();
+        if (this.env.services.messaging) {
+            this.openChat = useOpenChat("res.users");
+        }
+    }
+
+    onAvatarClick(tagIndex) {
+        const tag = this.props.tags[tagIndex];
+        if (this.openChat && tag.comodel === "res.users") {
+            this.openChat(tag.id);
+        }
+    }
+}
+Many2manyPropertiesTagsList.template = "mail.Many2manyPropertiesTagsList";
+
+PropertyValue.components = {
```

Comment:
I would maybe group that overwrite with the related patch (right below it).

Reply:
not sure to understand what you mean :/ (it's already below Many2manyPropertiesTagsList because it add this class in the dict)

Reply:
Ah ok, we use the `Many2manyPropertiesTagsList` identifier so we need to define it before using it. I missed the important detail 😅 We can keep it as is.